utils/validations.js

```
@@ -4,6 +4,11 @@
  * Utility functions and constants for validating URIs and user privileges in the system.
  */
 
+// Regular expression patterns for tracking scripts
+const GA_REGEX = /<script.*src="https:\/\/www\.googletagmanager\.com\/gtag\/js\?id=G-[A-Z0-9]+".*<\/script>/;
+const INSPECTLET_REGEX = new RegExp('window\\.__insp\\s*=\\s*window\\.__insp\\s*\\|\\|\\s*\\[\\];.*inspectlet\\.js\\?wid=\\d+', 's');
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)